tests/http/src/test/java/org/mule/test/module/http/functional/listener/HttpListenerCustomTlsConfigTestCase.java

```diff
@@ -20,11 +20,16 @@
 
   @Rule
   public DynamicPort port1 = new DynamicPort("port1");
+
   @Rule
   public DynamicPort port2 = new DynamicPort("port2");
+
   @Rule
   public DynamicPort port3 = new DynamicPort("port3");
 
+  @Rule
+  public DynamicPort port4 = new DynamicPort("port4");
```

Comment:
Looking at the tests, it seems those ports end up being override by the operation. So instead of this you could have a static port (for request configs it will have no effect) or just avoid the port attribute in the config.